D1673/P1673.md

```diff
@@ -6529,40 +6557,61 @@ private:
 
   * [2.3]{.pnum} `Accessor` meets the accessor policy requirements **[mdspan.accessor.reqmts]**.
 
+  * [2.4]{.pnum} _`scalable`_`<ScalingFactor, element_type>` is `true`. 
```

Comment:
I think this definition is circular, because the `element_type` of `scaled_accessor` necessarily comes from the product of the scaling factor and the nested accessor's element type.  Should we consider the following definition instead?

```suggestion
  * [2.4]{.pnum} _`scalable`_`<ScalingFactor, typename Accessor::element_type>` is `true`. 
```